src/containers/Authentication/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import Register from "../../components/Register";
+import Login from "../../components/Login";
+import { Form, Row, Button } from "antd";
+
+const layout = {
+  labelCol: { span: 4 },
+  wrapperCol: { span: 8 },
+};
+
+const tailLayout = {
+  wrapperCol: { offset: 8, span: 16 },
+};
+
+const Authentication = () => {
+  const [show, setShow] = useState(true);
+  const [click, setClick] = useState(false);
+
+  const handleShow = () => {
+    setShow(true);
+    setClick(false);
+  };
+
+  const handleClick = () => {
+    setClick(true);
+    setShow(false);
+  };
+
+  return (
+    <div>
+      <div>
+        <Form.Item style={{ marginBottom: "5em" }} {...layout} {...tailLayout}>
+          <Row>
+            <Button
```

Comment:
Can you make selected button's background gray like in Figma?

Reply:
done.